src/DatasetAPI/Datasets.jl

```diff
@@ -285,7 +285,7 @@ function toaxis(dimname, g, offs, len)
         catch
             ar[:]
         end
-        DD.Ti(tsteps[offs+1:end])
+        DD.rebuild(DD.name2dim(axname), tsteps[offs+1:end])
```

Comment:
Should we make the axis that we get here a subtype of Ti?

Reply:
no